tests/examples/test_examples.py

```diff
@@ -198,7 +201,9 @@ def _assert_snapshot(example, url, example_type):
 
     assert success, "%s failed to load" % example.relpath
     assert no_resources, "%s failed with %d missing resources" % (example.relpath, len(resources))
-    assert no_errors, "%s failed with %d errors" % (example.relpath, len(errors))
+
+    if example_type != "server":
```

Comment:
The issue is that all server examples end up with an error because of a race condition related to: https://github.com/bokeh/bokeh/blob/master/tests/plugins/phantomjs_screenshot.js#L85

I think it's a race condition where the document isn't evaluated and appended to the Bokeh variable before this line is evaluated, so we write an error. It's unclear to me what the best work-around is.

ping @mattpap